joss/paper.md

```diff
@@ -30,40 +30,40 @@ Consequently, these populations undergo evolution [@pennock2007models], and can
 One key benefit is complete, exact observability.
 For example, it is possible to perfectly record the full set of parent-child relationships over the history of a population, yielding precise and accurate phylogenies (ancestry trees).
-This information reveals the sequences of events behind gain, loss, or maintenance of specific traits, and also facilitates making inferences about the general factors driving evolution [@dolsonInterpretingTapeLife2020;@moreno2023toward].
+This information reveals the sequences of events behind gain, loss, or maintenance of specific traits, and also facilitates making inferences about the underlying mechanisms driving a given evolutionary system [mooers1997inferring;@dolsonInterpretingTapeLife2020;@moreno2023toward].
 
 The Phylotrack project provides libraries for tracking and analyzing phylogenies in *in silico* evolution.
 The project is composed of 1) Phylotracklib: a header-only C++ library, developed under the umbrella of the Empirical project [@ofria2020empirical], and 2) Phylotrackpy: a Python wrapper around Phylotracklib, created with Pybind11 [@pybind11].
 Both components supply a public-facing API to attach phylogenetic tracking to digital evolution systems, as well as a stand-alone interface for measuring a variety of popular phylogenetic topology metrics [@tuckerGuidePhylogeneticMetrics2017].
-The underlying algorithm design prioritizes efficiency, allowing Phylotrack to support large systems with rapid generational turnover.
-The underlying C++ implementation ensures fast, memory-efficient performance, with multiple explicit features (e.g., phylogeny pruning and abstraction, etc.) that facilitate reducing the memory footprint of phylogenetic information.
+The underlying algorithm design prioritizes efficiency, allowing Phylotrack to support large agent populations with rapid generational turnover.
+The underlying C++ implementation ensures fast, memory-efficient performance, with multiple explicit features (e.g., phylogeny pruning and abstraction, etc.) that can reduce the memory footprint of phylogenetic information.
 
 # Statement of Need
 
 <!-- was going to say a rich history of sophisticated phylogenetic analyses... -->
 *In silico* evolution work enjoys a rich history of phylogenetic record-keeping and analysis, and many systems facilitate tracking phylogenies [@ray1992evolution;@ofria2004avida;@bohm2017mabe;@de2012deap;@Garwood_REvoSim_Organism-level_simulation_2019].
 However, to our knowledge, no other general-purpose perfect phylogeny tracking library exists; prior work has used bespoke system- or framework-specific implementations.
-In contrast, Phylotrack provides ready-built tracking flexible enough to attach to any system containing replicating entities.
+In contrast, Phylotrack provides ready-built tracking flexible enough to attach to any simulation of replicating entities.
```

Comment:
How about "digital population of"? Since a lot of these aren't exactly simulations.

Reply:
I like that! could also go with "population of digital"